### PR TITLE
Changes in Sections enumeration

### PR DESCRIPTION
**Pull request by robtel**
After a short but productive discussion, it was decided to introduce changes in the document structure:

1. Section titles should start with a prefix phrase: Section and Roman numeral denoting sequential index of that section. 
    E.g., "SECTION I" or "SECTION II"
2. To better represent this change in HTML output, the headers of the documents should be numerated as well. 
    E.g., "1.1. STEP 1. Identify sections" 

Please, **provide your comments in the upcoming 10 days** since our QA-manager (@vanilalove) is expected to review and approve this Pull Request before our Repository Maintainer (@voltel) can merge it into the "main" branch of our document project.

### CODE REVIEW

File: source/section_one.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 =========================================
-SECTION I: STRUCTURING SPHINX PROJECT
```
Comment (voltel) [inline source/section_one.rst]: A colon after "SECTION I". Shouldn't we use a period "." instead?

File: source/section_one.rst [hunk 2]
```diff
@@ -28,7 +28,7 @@ It's fun and it's a team-building activity.
 
 
 STEP 2. Create ".rst" files
---------------------------------
```
Comment (voltel) [inline source/section_one.rst:31]: For headings of level-2 in this file we should use equal signs ("==="), not dashes ("----") which are used for level-3 headings here.

File: source/section_two.rst [hunk 3]
```diff
@@ -18,7 +18,7 @@ resolve merging conflicts when a new Pull Request
 has clashing edits with current text in the "main" branch.
 
 
-Step 2. Identify a team of document editors
```
Comment (voltel) [inline source/section_two.rst:21]: In other places, the whole word "STEP" is uppercase, here only the first letter is capitalized.

**Review by voltel — COMMENTED** [review]
I have made a new commit d3e34d618508e538bef598fcdc069801f0a3ff34  where I corrected some "formatting issues". 
Other than that, the proposed changes in this Pull Request are 👍 and the PR technically can be merged.

File: source/index.rst [hunk 4]
```diff
@@ -8,6 +8,7 @@ Welcome to Test Project's documentation!
 
 .. toctree::
    :maxdepth: 2
+   :numbered:
```
Comment (vanilalove) [inline source/index.rst:11]: 👍

File: source/section_one.rst [hunk 5]
```diff
@@ -1,5 +1,5 @@
 =========================================
-SECTION ONE: STRUCTURING SPHINX PROJECT
+SECTION I. STRUCTURING SPHINX PROJECT
```
Comment (vanilalove) [inline source/section_one.rst:2]: 🆗

File: source/section_one.rst [hunk 6]
```diff
@@ -28,7 +28,7 @@ It's fun and it's a team-building activity.
 
 
 STEP 2. Create ".rst" files
---------------------------------
+=============================
```
Comment (vanilalove) [inline source/section_one.rst:31]: Nice catch, @voltel! 💯

File: source/section_two.rst [hunk 7]
```diff
@@ -18,7 +18,7 @@ resolve merging conflicts when a new Pull Request
...
+STEP 2. Identify a team of document editors
```
Comment (vanilalove) [inline source/section_two.rst:21]: 👍

**Review by vanilalove — APPROVED** [review]
Thanks, @robtel  and @voltel! Nice job and fast! Way to go! 🥇 
Approve merge.

**Comment by vanilalove** [comment]
@voltel Proceed with the merge.

**Pull request merged**